src/ServicePulse.Host/app/js/views/failed_messages/controller.js

```diff
@@ -107,6 +107,27 @@
             vm.loadMoreResults(vm.selectedExceptionGroup);
         };
 
+        vm.viewMessage = function (message) {
+            $location.path(`/failed-messages/message/${message.id}`);
+        };
+
+        vm.retryMessage = function(message, $event) {
+            toastService.showInfo("Retrying the message " + message.id + " ...");
```

Comment:
Does the message id mean anything to the user? Is there a better value we can show here @sergioc ?

Reply:
There's no value in specifying message ID in the toast notification since that's the only message you're retrying. A generic message would suffice. Suggestion:

> Message retry requested